openbrews/recipe/add/hop.html

```diff
@@ -0,0 +1,40 @@
+<div class="list list-inset">
```

Comment:
There is the same issue with the top border on this inset list as found in #23. See below:
![image](https://cloud.githubusercontent.com/assets/4491428/19152039/fef61e12-8b9d-11e6-9339-828c2c0c4840.png)